Code/Tools/AssetProcessor/native/ui/MainWindow.cpp

```diff
@@ -801,42 +809,68 @@ void MainWindow::CheckAssetServerStates()
         ui->sharedCacheDiscardButton->setEnabled(false);
     }
 
-    switch (m_cacheServerData.m_errorLevel)
+    switch (m_cacheServerData.m_statusLevel)
     {
-        case CacheServerData::ErrorLevel::None:
+        case CacheServerData::StatusLevel::None:
         {
+            m_cacheServerData.m_updateStatus = true;
+            ui->sharedCacheStatus->setStyleSheet("background-color: transparent; color : white;");
```

Comment:
I have no idea what the standard is for Qt styling stuff, but is this the preferred way of setting styles?  Would it be better to put these settings in a .qss file and then switch them out?  Might need to discuss with someone from the editor team on best practices here.

Reply:
I am not sure how to align QT's CSS with the code, so I can try but the style sheet will need to change between status message types... so that is a twist too